plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -103,18 +103,18 @@
 import com.google.common.io.Files;
 
 /**
- * Manager class for ML models. It contains ML model related operations like upload, load model etc.
+ * Manager class for ML models. It contains ML model related operations like upload, load model
```

Comment:
Why this file is modified?

Reply:
my bad,the java format of this file is changed accidentally